app/assets/javascripts/analytics.js.erb

```diff
@@ -136,9 +136,7 @@ var linkedDomains = [
 ]
```

Comment:
We've also a conditional block at the top of this file: 
https://github.com/alphagov/static/blob/dea9d162882005b2205575a368d1b771744effcb/app/assets/javascripts/analytics.js.erb#L3-L5

that we'll need to remove otherwise this conditional won't work in replatforming and this will leave a lingering .erb file to trip someone up later. 

I wonder if the ga4-analytics can be a separate file where the ENV var conditional is in the HTML? @andysellick what do you think?

Reply:
We're currently working to remove this logic entirely, hopefully within the next few weeks. We were using this approach to get the new analytics up and running quickly on integration but we're now looking at doing it 'properly' i.e. integrating it with the cookie consent mechanism and doing it entirely in JS (no .erb required).